tests/DucksNet.IntegrationTests/EmployeesControllerTests.cs

```diff
@@ -52,8 +52,112 @@ public async void When_CreatedEmployee_Then_ShouldReturnEmployeeInTheGetRequest(
             employee.OwnerPhone.Should().Be(sut.OwnerPhone);
         }
     }
+    [Fact]
```

Comment:
These tets are fine but you're missing a bunch of negative cases for: get, post, delete, update personal information and there is no test for update office.